keras/src/backend/openvino/linalg.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import scipy.linalg as sl
+
+from keras.src.backend import standardize_dtype
+from keras.src.backend.common import dtypes
+from keras.src.backend.openvino.core import convert_to_tensor
+
+
+def cholesky(a):
+    return np.linalg.cholesky(a)
```

Comment:
All of the code in many of these files still refers to numpy. If something isn't implemented, it's better to raise `NotImplementedError`, so we can keep track of it.

Reply:
Done.